test/column/colSpan.test.ts

```diff
@@ -149,23 +148,28 @@ describe('colSpan', () => {
 
   it('should scroll to the merged cell when selected', () => {
     setupColSpanGrid(30);
-    const grid = screen.getByRole('grid');
     userEvent.click(getCellsAtRowIndex(8)[23]); // last visible cell (1920/80)
+    const spy = jest.spyOn(window.HTMLElement.prototype, 'scrollIntoView');
     navigate(3);
-    expect(grid.scrollLeft).toBe(240);
+    expect(spy).toHaveBeenCalled();
+    spy.mockReset();
```

Comment:
We could check that the number of calls is incremented instead of resetting each time, up to you
```suggestion
    expect(spy).toHaveBeenCalledTimes(1);
```